.gitpod.yml

```diff
@@ -0,0 +1,18 @@
+image:
+  file: .gitpod.Dockerfile
+github:
```

Comment:
```suggestion
task:
  - name: install current state of nf-core/tools
    init: |
       pip install --upgrade -r requirements-dev.txt -e .
github:
```

Reply:
Is the one installed via conda out of date or something? 

Or does this need to be run at the beginning of each start up ( Or can I put this in the Dockerfile )?

Reply:
Since this is the tools repo, I think it is best if the tools there is based on the current state of the repo. So if you want to test a change in the repo, you can do it straight from this gitpod.

Reply:
So just to be clear this should only be done for the tools repo and not the one propagated via the pipeline template?

Reply:
exactly. for the pipeline template the conda version should be used.